src/platform/ios/mod.rs

```diff
@@ -489,6 +489,8 @@ fn create_delegate_class() {
             let view: id = msg_send![view_class, alloc];
             let view: id = msg_send![view, initForGl:&bounds];
 
+            msg_send![view_controller, setView: view];
```

Comment:
You need to prefix this with `let _: () = ` like in below, since the stabilization of `!` (the never type) will change the type inference from `()` to `!`, resulting in sigsegv/sigill.

I also prefer `setView:view`, since AFAIK it's more consistent with conventional Obj-C style, though correct me if I'm wrong on that.